mod/run/shaderpacks/DefaultShaderpack/shaders/gui.frag

```diff
@@ -59,12 +59,12 @@ layout(location = 0) out vec4 color_out;
 
 void main() {
     if(textureSize(colortex, 0).x > 0) {
-        vec4 tex_sample = texture(colortex, uv);
+        vec4 tex_sample = texture(colortex, mod(uv,1.0));
```

Comment:
The texture coords should already be in the 0 - 1 range though... right?

Reply:
I think someone added it so that the menu background is actually tiled, and I think the proper way to do that would be to actually make the textures tiled when exceeding uv 0-1, instead of clamping the value (which I'm pretty sure is an option)